.github/workflows/integration-tests.yml

```diff
@@ -107,6 +73,27 @@ jobs:
             ccip_changes:
               - '**/*ccip*'
               - '**/*ccip*/**'
+      - name: Filter not ignored changes
+        uses: ./.github/actions/filter-changes
+        id: not_ignored_changes
+        with:
+          # Filter files that match **every** pattern in this list
+          every-filter: |
+            - '**'                     # Initially include all files
+            - '!**/*.md'               # Exclude all markdown files (lowercase)
+            - '!**/*.MD'               # Exclude all markdown files (uppercase)
```

Comment:
I think the same logic applies here for integration-tests/.
```suggestion
```

Reply:
@jmank88 these are end-end tests. Is there any reason we should run them on *.md changes, even if the markdown is generated? I updated this PR to only run unit tests in this case.